.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,112 @@
+# Based on the analogous pypa/setuptools form.
+---
+name: 🐜 Bug report
+description: >-
+  Create a report to help us improve when
+  something is not working correctly.
+title: 'Bug: '
+labels:
+- bug
+
+body:
+- type: markdown
+  attributes:
+    value: >
+      **Thank you for reporting a bug in XLConnect!**
+      
+
+      ⚠
+      Verify first that your issue is not
+      [already reported on GitHub][issue search].
+
+      Also test if the latest CRAN release is affected.
+
+      If you are seeking community support, please consider
+      [asking a question on stackoverflow][stackoverflow].
+
+
+
+      Thank you for your collaboration!
+
+
+      [stackoverflow]: https://stackoverflow.com/questions/tagged/xlconnect
+
+      [issue search]: https://github.com/miraisolutions/xlconnect/search?q=is%3Aissue&type=issues
+
+- type: markdown
+  attributes:
+    value: >-
+      **Environment**
+- type: textarea
+  attributes:
+    label: sessionInfo() output
+    description: >-
+      Output of running sessionInfo in your r session with XLConnect loaded
+    placeholder: >-
+      R version 4.2.2 (2022-10-31 ucrt)
+      Platform: x86_64-w64-mingw32/x64 (64-bit)
+      Running under: Windows 10 x64 (build 19044)
+
+      Matrix products: default
+
+      locale:
+      [1] LC_COLLATE=English_United States.utf8
+      [2] LC_CTYPE=English_United States.utf8
+      [3] LC_MONETARY=English_United States.utf8
+      [4] LC_NUMERIC=C
+      [5] LC_TIME=English_United States.utf8
+
+      attached base packages:
+      [1] stats     graphics  grDevices utils     datasets  methods   base
+
+      other attached packages:
+      [1] XLConnect_1.0.7
+
+      loaded via a namespace (and not attached):
+      [1] compiler_4.2.2 rJava_1.0-6
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Additional environment information
+    description: >-
+      Feel free to add more information about your environment here.
+
+- type: textarea
+  attributes:
+    label: Description
+    description: >-
+      A clear and concise description of what the bug is.
+    placeholder: >-
+      I tried doing X and I expected it to result in Y because the docs
+      mentioned Z but what happened next what totally unexpected!
+      And here's why...

```

Comment:
```suggestion
      I tried doing X and I expected it to result in Y because the docs
      mentioned Z but what happened next was totally unexpected!
      And here's why...

```